### DISCUSSION TITLE
Getting [Github] Addon into The Catalog

### DISCUSSION THREAD

**Discussion by Ironlenny** [ogri-la/wowman#386] [Q&A] [answered]
I have a simple addon that I'm uploading to Github and I want to make sure it shows up in Strongbox's catalog. What steps do i need to take to ensure that?

**Comment by torkus** [comment]
Strongbox uses the catalogue compiled by @layday here:
https://github.com/layday/github-wow-addon-catalogue

IIRC you won't have to do much to get your addon detected and included, but a few things may exclude you, like not using Github's *releases* feature.

**Comment by layday** [comment, marked as answer]
That’s right, you need to attach the addon zip to the release, the addon folder must be at the top level inside the zip file, and the repository needs to be discoverable, e.g. by tagging it with “wow-addon”.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by Ironlenny** [reply]
I understand how to tag releases, but how do I tag the repo with "wow-addon"?

**Reply by torkus** [reply]
For me and tagging the 'strongbox' repo, there is a gear icon on the main page next to the 'About' header on the right. Clicking that gets me to a place where I can add 'Topics', which is what we mean by tags:

![Screenshot at 2023-03-28 18-14-11](https://user-images.githubusercontent.com/48790775/228165878-86c06b4d-0de0-4b91-98cb-bbaae588634c.png)
